src/components/upload-menu-item/UploadMenuItem.tsx

```diff
@@ -60,15 +65,19 @@ export const UploadMenuItem: React.FC<IUploadMenuItem> = ({ ...props }) => {
   };
 
   return (
-    <SUploadMenuItemWrapper hovering={hovering} active={active} isDefault={!props.state || props.state === "default"}>
+    <SUploadMenuItemWrapper
+      hovering={hovering}
+      activated={activated}
+      isDefault={!props.state || props.state === "default"}
+    >
       <SUploadMenuItem
         onMouseEnter={() => setHovering(true)}
         onMouseLeave={() => {
           setHovering(false);
-          setActive(false);
+          setActivated(false);
         }}
-        onMouseDown={() => setActive(true)}
-        onMouseUp={() => setActive(false)}
+        onMouseDown={() => setActivated(true)}
```

Comment:
that would be okey somewhere in the story but to ship this with the component would not be wanted in league

i do not want to ship the onMouse... handlers here  
instead just passing the props `activated` from the storybook should be enough 
the hover is okey to do with css only and also no onMouse... needed

Reply:
Perfect!